cc/mcts_tree.cc

```diff
@@ -435,6 +452,8 @@ void MctsTree::BackupValue(MctsNode* leaf, float value) {
 
 void MctsTree::InjectNoise(const std::array<float, kNumMoves>& noise,
                            float mix) {
+  MG_DCHECK(root_->is_expanded);
```

Comment:
We can't expand the root here any more because it required performing an inference (which happens without the `InjectNoise` caller being aware of it); concurrent_selfplay requires that all inferences are performed explicitly.